src/visualizers/Histogram.ts

```diff
@@ -1,8 +1,12 @@
+import * as brush from 'p5.brush'
+
 import {VisualizerExportModule} from './VisualizerInterface'
-import {P5Visualizer} from './P5Visualizer'
-import {ParamType} from '../shared/ParamType'
-import type {GenericParamDescription, ParamValues} from '../shared/Paramable'
-import type {SequenceInterface} from '../sequences/SequenceInterface'
+import {P5GLVisualizer} from './P5GLVisualizer'
```

Comment:
So, I'm accidentally moving the histogram in 3D with my mouse when I load and play with it, and then I can't reset it except by resetting the visualizer.  Do you think this is annoying?  I think in the case of the histogram visualizer, maybe we want the 3d mouse control turned off by default.  Does anyone really want to rotate the 2d graph in 3d for some purpose?

Reply:
No, this was just me being lazy and using the default camera controls that come with p5 WebGL mode. The latest commit implements more useful controls for typical (2D) visualizers: left drag pans, mouse wheel zooms.